docker/compose/demo/compose.yml

```diff
@@ -97,7 +97,8 @@ services:
       - DATAVERSE_URL=http://dataverse:8080
       - TIMEOUT=10m
       - PREVIEWERS_PROVIDER_URL=http://${MACHINE_IP:-localhost}:9000
```

Comment:
Should we consider removing this MACHINE_IP stuff from this compose file? We already removed it from the dev version.

Reply:
I expect that it would make a lot of sense to be able to provide an alternative IP address (or host name) in a production environment. Leaving it in would be a good idea IMHO.

Reply:
Yeah, I suspect we'll dig into this more when we work on fronting Dataverse with some sort of proxy:

- #10359